### PR TITLE
test code review

### PR DESCRIPTION
**Pull request by BrandonWu500**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -8,7 +8,8 @@ const Container = styled.div`
 const Home = () => {
   return (
     <Container theme="dark">
-      <h1>Homess</h1>
+      <h1 className="test">Homes</h1>
```
Comment (brandonwu501) [inline pages/index.tsx]: I think you can remove the `className` prop

Comment (brandonwu501) [inline pages/index.tsx]: Maybe change this to just "Home"

**Review by brandonwu501 — CHANGES_REQUESTED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (BrandonWu500) [reply pages/index.tsx]: Nice catch!

Comment (BrandonWu500) [reply pages/index.tsx]: test

Comment (brandonwu501) [reply pages/index.tsx]: test

**Review by brandonwu501 — APPROVED** [review]
Looks good!

**Pull request merged**